controls/diffutils_works.rb

```diff
@@ -0,0 +1,47 @@
+title 'Tests to confirm diffutils works as expected'
+
+plan_origin = ENV['HAB_ORIGIN']
+plan_name = input('plan_name', value: 'diffutils')
+
+control 'core-plans-diffutils-works' do
+  impact 1.0
+  title 'Ensure diffutils works as expected'
+  desc '
+  Verify diffutils by ensuring 
+  (1) its installation directory exists and 
+  (2) that it returns the expected version.
+  (3) should detect equal contents
+  (4) should detect different content
+  '
+  
+  plan_installation_directory = command("hab pkg path #{plan_origin}/#{plan_name}")
+  describe plan_installation_directory do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not be_empty }
+    its('stderr') { should be_empty }
+  end
+  
+  command_relative_path = input('command_relative_path', value: 'bin/diff')
+  command_full_path = File.join(plan_installation_directory.stdout.strip, "#{command_relative_path}")
+  plan_pkg_version = plan_installation_directory.stdout.split("/")[5]
+  describe command("#{command_full_path} --version") do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not be_empty }
+    its('stdout') { should match /diff \(GNU diffutils\) #{plan_pkg_version}/ }
+    its('stderr') { should be_empty }
+  end
+
+  alpha_file = "/hab/svc/diffutils/config/fixtures/alpha"
```

Comment:
Instead of shipping these files and having to hardcode paths, could you rework this using a one-liner such as the below?
```
[10][default:/src:2]# diff  <(echo "alpha" ) <(echo "beta")
1c1
< alpha
---
> beta
```

Reply:
Done. Thanks @skpaterson.

Reply:
Hi @skpaterson.  As the inline functional tests were failing, I (1) restored the original tests using fixture files and (2) raised an issue #2.

Reply:
Hi @skpaterson: As discussed, have used azuredevops local package build pipeline and the above inline verification is now passing ok in CI.  Also, however, raised an issue https://github.com/chef-base-plans/plan-builder/issues/20